test/k8sT/Policies.go

```diff
@@ -1661,19 +1677,35 @@ var _ = Describe("K8sPolicyTest", func() {
 				})
 
 				It("Validates fromEntities remote-node policy", func() {
-					By("Installing default-deny ingress policy")
-					importPolicy(kubectl, testNamespace, cnpDenyIngress, "default-deny-ingress")
-
-					By("Checking that remote-node is disallowed by default")
-					validateNodeConnectivity(HostConnectivityAllow, RemoteNodeConnectivityDeny)
+					installDefaultDenyIngressPolicy()
 
 					By("Installing fromEntities remote-node policy")
 					importPolicy(kubectl, testNamespace, cnpFromEntitiesRemoteNode, "from-entities-remote-node")
 
 					By("Checking policy correctness")
-					validateNodeConnectivity(HostConnectivityAllow, RemoteNodeConnectivityAllow)
+					validateConnectivity(HostConnectivityAllow, RemoteNodeConnectivityAllow, PodConnectivityDeny)
 				})
 			})
+
+			It("Validates fromEntities cluster policy", func() {
+				installDefaultDenyIngressPolicy()
+
+				By("Installing fromEntities cluster policy")
+				importPolicy(kubectl, testNamespace, cnpFromEntitiesCluster, "from-entities-cluster")
+
+				By("Checking policy correctness")
+				validateConnectivity(HostConnectivityAllow, RemoteNodeConnectivityAllow, PodConnectivityAllow)
+			})
+
+			It("Validates fromEntities all policy", func() {
+				installDefaultDenyIngressPolicy()
+
+				By("Installing fromEntities cluster policy")
+				importPolicy(kubectl, testNamespace, cnpFromEntitiesAll, "from-entities-all")
+
+				By("Checking policy correctness")
+				validateConnectivity(HostConnectivityAllow, RemoteNodeConnectivityAllow, PodConnectivityAllow)
```

Comment:
I'd expect the policy correctness test to be a bit different for the two cases to pick up the policy difference between `fromEntities` `cluster` and `fromEntities` `all`.

Reply:
OK, I'll add "ingress connectivity from world" test case.

Reply:
You can use the third node, which runs without Cilium for that. See e.g. `testCurlFromOutside`. You'll need to condition that part of the test on `helpers.ExistNodeWithoutCilium()`.

To run something equivalent locally, you can use:
```
NO_CILIUM_ON_NODE="k8s3" K8S_NODES=3 NETNEXT=1 KUBEPROXY=0 K8S_VERSION=1.13 ginkgo -v --focus="K8sPoli" -- -cilium.provision=true -cilium.testScope="k8s" -cilium.holdEnvironment=true
```